src/AppInstallerCLITests/TestData/Manifest-Good-SystemReferenceComplex.yaml

```diff
@@ -4,14 +4,10 @@ Version: 1.7.32
 Publisher: Microsoft
 License: MIT License
 Description: A complex system reference test file.
-InstallerType: Zip
+InstallerType: EXE
```

Comment:
Nit: You use inconsistent case (EXE vs exe) on these

Reply:
I changed all the installer types in this test manifest to lowercase to match other yamls.